src/inspect_ai/tool/_tools/_computer/_computer.py

```diff
@@ -0,0 +1,200 @@
+import asyncio
+import shlex
+from typing import Awaitable, Callable
+
+from inspect_ai.model import Content
+from inspect_ai.tool import Tool, tool
+from inspect_ai.tool._tools._computer._computer_common import (
+    cursor_position,
+    double_click,
+    left_click,
+    left_click_drag,
+    middle_click,
+    mouse_move,
+    press_key,
+    right_click,
+    screenshot,
+    type,
+)
+
+# Export ToolResult from inspect_ai.tool
+ToolResult = str | int | float | bool | list[Content]
+
+ActionFunction = Callable[[str], ToolResult | Awaitable[ToolResult]]
+
+
+@tool(parallel=False)
+def computer(timeout: int | None = None) -> Tool:
+    """
+    Computer interaction tool.
+
+    Use a mouse and keyboard to interact with a computer, and take screenshots.
```

Comment:
These doc comments I believe are not played back to the model (rather just the doc comments for `execute`) so we can shorten these up and make them more developer directed.